### PR TITLE
AppsPlugin: add switch to ignore XDG OnlyShowIn Desktop Attr

### PR DESCRIPTION
**Pull request by UffeJakobsen**
AppsPlugin: add switch to ignore XDG OnlyShowIn Desktop Attr

Example:
On my system (Arch Linux) the excellent file manager Caja is flagged as show only in MATE - it works perfectly on other desktop environments

The new switch will make such desktop apps available by ignoring that attr

### CODE REVIEW

**Comment by otsaloma** [comment]
I guess this is needed, I remember seeing complaints about this with some other launchers.

File: catapult/plugins/apps.py [hunk 1]
```diff
@@ -18,13 +18,38 @@
...
 from catapult.i18n import _
 from gi.repository import Gio
+from gi.repository import Gtk
+
+
+class AppsPluginPrefs(PreferencesItem):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.label = Gtk.Label(label=_("Ignore OnlyShowIn Attr"))
```
Comment (otsaloma) [inline catapult/plugins/apps.py:32]: The label is a bit overly technical, how about

```python
desktop = catapult.util.get_desktop_environment() or _("native")
label = _("Show non-{} apps").format(desktop)
self.label = Gtk.Label(label=label)
```

File: catapult/plugins/apps.py [hunk 2]
```diff
@@ -18,13 +18,38 @@
...
+        self.widget = Gtk.Switch()
+        self.widget.connect("notify::active", self._on_widget_notify_active)
```
Comment (otsaloma) [inline catapult/plugins/apps.py:34]: I guess this is some testing leftover? Please remove this line and the empty callback method.

File: catapult/plugins/apps.py (hunk 1 above)
Comment (UffeJakobsen) [reply catapult/plugins/apps.py]: Ok - attempting new push

File: catapult/plugins/apps.py (hunk 2 above)
Comment (UffeJakobsen) [reply catapult/plugins/apps.py]: Ok - attempting new push

**Comment by otsaloma** [comment]
Thanks, merging!

**Pull request merged**